src/Debugging-Core/Context.extension.st

```diff
@@ -55,6 +55,23 @@ Context >> namedTempAt: index put: aValue [
 	^self debuggerMap namedTempAt: index put: aValue in: self
 ]
 
+{ #category : #'*Debugging-Core' }
+Context >> pcRangeContextIsActive: contextIsActive [
```

Comment:
Method naming seems wrong, seems its supposed to return a boolean.
However I the next commit contains a fix, so it should be accepted as is.